services/user.py

```diff
@@ -0,0 +1,44 @@
+from typing import Optional
+
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(username: str,
+                password: str,
+                email: Optional[str] = "",
+                first_name: Optional[str] = "",
+                last_name: Optional[str] = "") -> None:
```

Comment:
You have to use Optional only if your variable can be None. Your variables can't be None, so there are not optional